src/utils/utils.ts

```diff
@@ -31,9 +31,26 @@ function hasSprykerCertifications(certifications: SprykerCertifications): boolea
   return certifications.backEndDeveloper || certifications.solutionArchitect;
 }
 
+// Generate URL-friendly slug from first and last name
+function generateNameSlug(firstName: string, lastName: string): string {
```

Comment:
for next time a smal suggestion: `generateSlugByFirstnameLastname()`
the current name does not really tell what it is about; good function/method names are better than comments